temp/legacy-code/src/main/java/com/ivy/legacy/ui/component/transaction/TransactionCard.kt

```diff
@@ -397,7 +401,7 @@ private fun TransactionBadge(
             .background(backgroundColor, UI.shapes.rFull)
             .clickable {
                 onClick()
-            },
+            }.padding(end = 10.dp),
```

Comment:
Remove this padding and increase the last spacer from `20.dp` to some more. Also why do we add more padding? What's the benefit?

Reply:
The purpose of this padding is to ensure that the text within the blue transaction badge does not extend beyond its borders.
<img width="1165" alt="Screenshot 2024-06-20 alle 22 42 02" src="https://github.com/Ivy-Apps/ivy-wallet/assets/78317934/fe1b24f2-9994-4e73-938c-234dc6ab7688"> 
Unfortunately your suggestion doesn't work. Instead with the padding the result is: 
<img width="1229" alt="Screenshot 2024-06-20 alle 22 46 27" src="https://github.com/Ivy-Apps/ivy-wallet/assets/78317934/95462e7d-8267-4bae-84ed-bc2e2ff7aaeb">

Reply:
Good, makes sense! 💯